lib/KBox/src/KMessage.h

```diff
@@ -157,20 +164,20 @@ class IMUMessage: public KMessage {
     void accept(KVisitor &v) const {
       v.visit(*this);
     };
-
+    
```

Comment:
What I meant is that there are a few white spaces at the beginning of an empty line (github highlights them). Not a big deal, I will take care of that.